src/wp-includes/comment-template.php

```diff
@@ -1034,15 +1034,21 @@ function comment_text( $comment_ID = 0, $args = array() ) {
  * Retrieves the comment time of the current comment.
  *
  * @since 1.5.0
+ * @since 4.4.0 Added the ability for `$comment_ID` to also accept a WP_Comment object.
  *
- * @param string $format    Optional. PHP time format. Defaults to the 'time_format' option.
- * @param bool   $gmt       Optional. Whether to use the GMT date. Default false.
- * @param bool   $translate Optional. Whether to translate the time (for use in feeds).
- *                          Default true.
+ * @param string         $format     Optional. PHP date format. Defaults to the 'time_format' option.
+ * @param bool           $gmt        Optional. Whether to use the GMT date. Default false.
+ * @param bool           $translate  Optional. Whether to translate the time (for use in feeds).
+ *                                   Default true.
+ * @param int|WP_Comment $comment_ID WP_Comment or ID of the comment for which to get the date.
```

Comment:
```suggestion
 * @param int|WP_Comment $comment_ID Optional. WP_Comment or ID of the comment for which to get the date.
                                     Default is 0, or the global comment.
```
Just a quick note that this parameter is optional and should be documented as such. 🙂

Reply:
@DSGND ☝️